docker/Makefile

```diff
@@ -3,25 +3,22 @@ VERSION = 0.4.7
 
 TAG1 = us.gcr.io/broad-dsp-lrma/$(IMAGE_NAME):$(VERSION)
 TAG2 = us.gcr.io/broad-dsp-lrma/$(IMAGE_NAME):latest
-TAG3 = us.gcr.io/broad-dsp-lrma/$(IMAGE_NAME):$(VERSION)
-TAG4 = us.gcr.io/broad-dsp-lrma/$(IMAGE_NAME):latest
 
 all: | build push
 
 build:
 	eval `ssh-agent`; \
 	ssh-add ~/.ssh/id_rsa;\
-	docker build --ssh default --squash -t $(TAG1) -t $(TAG2) -t $(TAG3) -t $(TAG4) . ;\
+	docker build --ssh default --squash -t $(TAG1) -t $(TAG2) . ;\
```

Comment:
I think to build the Docker image, I had to simplify some of this... something about Docker experimental features that I didn't take the time to read about.  So out of curiosity, what *is* going on here with the `--ssh agent` stuff? Is this to make the Longbow github repo clonable inside the Docker image?  Would just cloning from the https address rather than over ssh allow us to simplify this?

Reply:
Yeah - this requires experimental features to work.  It wasn't very hard to enable them on my VM, but it would be nice to not need them.

I don't remember exactly what the issue was, but that may have been it.  I'll do some tests to see if reverting to `https` will work.